docs/visualize/vega.asciidoc

```diff
@@ -16,21 +16,20 @@ NOTE: In Vega it is possible to load data dynamically, e.g. by setting signals a
 [[vega-quick-demo]]
 === Getting Started with Vega
 
-Follow these steps to create your first Vega visualization.
-
-. In Kibana, choose Visualize, and add Vega visualization. You should see a default graph.
-. Try changing `mark` from `line` to `point`, `area`, `bar`, `circle`,
-`square`, ... (see
-https://vega.github.io/vega-lite/docs/mark.html#mark-def[docs])
-. Try other https://vega.github.io/vega/examples/[Vega] or
-https://vega.github.io/vega-lite/examples/[VegaLite] visualizations. You
-may need to make URLs absolute, e.g. replace
+* Go to *Visualize*, click *+*, and select *Vega* to see an example graph.
```

Comment:
Not sure how to phrase this:  many users get confused by the fact that you can use two different languages - Vega and Vega-Lite when writing visualisations here. The default one that shows for everyone is Vega-Lite, but user can write in either of the languages. They are distinguished by the `schema` value at the top of the graph.  Any ideas on how/where to add that here?

Reply:
Maybe something like this?
<img width="856" alt="screen shot 2019-02-15 at 1 46 22 pm" src="https://user-images.githubusercontent.com/13408456/52877514-3211f000-3128-11e9-95ef-f24cf4a47a0b.png">